src/ui/SimpleFilter/SimpleFilter.ts

```diff
@@ -0,0 +1,188 @@
+import {l} from "../../strings/Strings";
+import {Assert} from "../../misc/Assert";
+import * as _ from 'underscore';
+import {Checkbox} from "../FormWidgets/Checkbox";
+import {IGroupByRequest} from "../../rest/GroupByRequest";
+import {QueryBuilder} from "../Base/QueryBuilder";
+import {IStringMap} from "../../rest/GenericParam";
+import {Utils} from "../../utils/Utils";
+import {BreadcrumbEvents, IClearBreadcrumbEventArgs, IPopulateBreadcrumbEventArgs} from "../../events/BreadcrumbEvents";
+import {BreadcrumbValueElement} from "../Facet/BreadcrumbValueElement";
+import {BreadcrumbValueList} from "../Facet/BreadcrumbValuesList";
+
+
+export interface ISimpleFilterOptions{
+    title: string;
+    values: string[],
+    field: IFieldOption,
+    //includeInBreadcrumb: boolean,
+    valueCaption?: any
+}
+
+export class SimpleFilter extends Component {
+    static ID = 'SimpleFilter';
+    static doExport = () => {
+        exportGlobally({
+            'SimpleFilter': SimpleFilter
+        })
+    };
+    static options: ISimpleFilterOptions = {
+
+        values: ComponentOptions.buildListOption<string>(),
+        field: ComponentOptions.buildFieldOption({ required: true, groupByField: true, section: 'Identification' }),
+        title: ComponentOptions.buildStringOption({defaultValue: l('NoTitle')}),
+        //includeInBreadcrumb: ComponentOptions.buildBooleanOption({defaultValue : true}),
+        valueCaption: ComponentOptions.buildCustomOption<IStringMap<string>>(() => {
+            return null;
+        }),
+    };
+
+    public checkboxes: {checkbox: Checkbox, value: string} [] =  [];
+    public checkboxContainer: Dom;
+    private expanded = false;
+
+    constructor(public element: HTMLElement, public options: ISimpleFilterOptions, public bindings?: IComponentBindings){
+        super(element, SimpleFilter.ID, bindings);
+        this.type = 'SimpleFilter';
+        this.options = ComponentOptions.initComponentOptions(element, SimpleFilter, options);
+
+        let select = $$('select', {className: 'select'});
+        let option = $$('option', {className: 'option'});
+        option.el.innerText = l(this.options.title);
+        select.el.appendChild(option.el) ;
+
+        let overselect = $$('div', {className: 'overselect'});
+        const multiselect = $$('div', { className: 'multiselect'});
+        multiselect.el.appendChild(select.el);
+        multiselect.el.appendChild(overselect.el);
+        this.element.appendChild(multiselect.el);
+
+        this.checkboxContainer = $$('div', { className: 'checkboxes'});
+        this.checkboxes = _.map(this.options.values, (value)=> this.createCheckboxes(value));
+        _.each(this.checkboxes, (result) => {
+            this.checkboxContainer.el.appendChild(result.checkbox.getElement())
+        });
+        this.element.appendChild(this.checkboxContainer.el);
+
+        $$(document.body).on('click', (e: Event)=> {
+            let target = <Element>e.target;
```

Comment:
This is way too complex.

* prevent event bubbling on your element. (e.stopPropagation)
* onRootClick -> close.